omc3/model/accelerators/lhc.py

```diff
@@ -379,34 +415,46 @@ def get_base_madx_script(self, best_knowledge: bool = False) -> str:
         high_beta = False
         madx_script = (
             f"{self._get_madx_script_info_comments()}"
-            f"! ----- Calling Sequence and Optics -----\n"
             f"call, file = '{self.model_dir / MACROS_DIR / GENERAL_MACROS}';\n"
             f"call, file = '{self.model_dir / MACROS_DIR / LHC_MACROS}';\n"
-            )
+        )
+        madx_script += f"omc3_beam_energy = {self.energy};\n"
+        madx_script += "exec, define_nominal_beams();\n\n"
         if self._uses_run3_macros():
-            LOGGER.debug("According to the optics year, Run 3 versions of the macros will be used")
+            LOGGER.debug(
+                "According to the optics year, Run 3 versions of the macros will be used"
+            )
             madx_script += (
                 f"call, file = '{self.model_dir / MACROS_DIR / LHC_MACROS_RUN3}';\n"
             )
 
-        madx_script += (
-            f"{self.load_main_seq_madx()}\n"
-            f"exec, define_nominal_beams();\n"
-        )
+        madx_script += "! ----- Calling Sequence and Optics -----\n"
+        madx_script += "option, -echo;  ! suppress output from base sequence loading to keep the log small\n"
+        madx_script += self.load_main_seq_madx()
+        madx_script += "\n\n"
+
```

Comment:
I am not sure. But wasn't this because of the bug in `acc-models` that the modifiers overwrite `NRJ`, making the `beam` command (which is used in `define_nominal_beams()` inconsistent?

Reply:
Ah yes, there was something like this. I think there was either a new file called or some correction by Riccardo in the acc-models-lhc scripts to get this of this re-definition some time ago. 

This way (as in: in the code of this PR right now) works, so I'm ok with that. Feel free to resolve this conversation or make a comment in the code or whatever you feel is best.